StereoKitC/systems/render.cpp

```diff
@@ -86,6 +86,23 @@ struct render_screenshot_t {
 	int32_t height;
 	float   fov_degrees;
 };
+struct render_screenshot_capture_t {
```

Comment:
I'd probably also suggest merging `render_screenshot_viewpoint_t` and `render_screenshot_t` together too! Instead of saving directly to file, a built-in function can be automatically submitted saved in the callback that will write to file?

Reply:
Sweet, will do! That'll look much cleaner. That's a great idea, I'll just assign it to something that sends it off to stbi :)